ballerina/client.bal

```diff
@@ -13,3 +13,2824 @@
 // KIND, either express or implied.  See the License for the
 // specific language governing permissions and limitations
 // under the License.
+
+import shopify.oas;
+
+import ballerina/http;
+
+# This is the public Shopify REST API.
+public isolated client class Client {
+    final oas:Client generatedClient;
+    final string accessToken;
+    final string apiVersion;
+
+    # Gets invoked to initialize the `connector`.
+    #
+    # + config - The configurations to be used when initializing the `connector` 
+    # + serviceUrl - URL of the target service 
+    # + return - An error if connector initialization failed 
+    public isolated function init(string accessToken, string apiVersion, ConnectionConfig config, string serviceUrl) returns error? {
+        self.accessToken = accessToken;
```

Comment:
I think the logic related to accessToken can be moved to security schema as an API key config in the OpenAPI spec(part of sanitization).

Reply:
Fixed in [bf17c6e](https://github.com/ballerina-platform/module-ballerinax-shopify/pull/7/commits/bf17c6e70cd5d849cf177606c8a4b4a4fcede997)